### PR TITLE
Add hello microverse

### PR DESCRIPTION
**Pull request by Blessman-Newton**
**Task: Hello Microverse**

_**I have ensured that:**_

1. There are no linter errors in the project.
2. Used Correct git-flow.
3.  Included a descriptive Readme file.
4. Used descriptive commit messages.
5. Followed the recommended HTML and CSS.
6. Documented my work in a professional manner according to Professional code
7.  Add node_modules to gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,163 @@
...
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
```
Comment (OyePriscilla) [inline README.md]: - [ ]  Please ensure that your `live demo link` rediects visitors to the appropraite page, if you don't have a live demo link at the moment kindly remove the header.

File: README.md [hunk 2]
```diff
@@ -1 +1,163 @@
...
+  <ul>
+    <li><a href="https://CSS.org/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
```
Comment (OyePriscilla) [inline README.md]: - [ ] The response in your key features are not engaging enough, kindly think of the key features (e.g. Add html file, set-up linters ....) you can add to your project. 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,163 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```
Comment (OyePriscilla) [inline README.md]: - [ ] The response in your  feature features headers are not engaging enough, kindly think of the feature features (e.g. Implement nav items, add images ...) you can add to your project. 👍

File: README.md [hunk 4]
```diff
@@ -1 +1,163 @@
...
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (OyePriscilla) [inline README.md]: - [ ] For your README file to be more professional, kindly remove this placeholder. It's there to eplain how you can add a license file. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Blessman-Newton 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
GitHub flow observed ✔️
Ignonored `node_modules` and `test.md` files as required ✔️
Added `index.html` and `style.css` files ✔️
Linters checks passed ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Blessman-Newton, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Customized Key Features Section. ✔️ 
- Customized Future Features Section. ✔️ 
- Removed placeholder content for MIT license ✔️ 
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**